src/System.Management.Automation/minishell/api/RunspaceConfigurationTypeException.cs

```diff
@@ -20,12 +20,7 @@ namespace System.Management.Automation.Runspaces
     ///     2. ISerializable
     /// -->
     [Serializable]
-#if CORECLR
-    internal
-#else
-    public
-#endif
-    class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
+    internal class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
```

Comment:
For PRs  #4447, #4446, #4445 and this one (#4448), can you please group them in a single PR?
You will need to do merge/rebase in one branch against other 3 branches, then you will have all those changes in one branch.